src/data_utils/preprocessing_helper.py

```diff
@@ -10,29 +10,22 @@
 with open(PRIORITIZE_ABBREVIATIONS_DIR) as f:
     PRIORITIZE_ABBREVIATIONS = json.load(f)
 
-
 def get_abbreviations():
     return CONTRACTIONS
 
-
 def get_prioritize_abbreviations():
     return PRIORITIZE_ABBREVIATIONS
 
-
 def get_contractions():
     return __get_contractions(CONTRACTIONS)
 
-
 def get_priortize_contractions():
     return __get_contractions(PRIORITIZE_ABBREVIATIONS)
 
-
```

Comment:
@LeviIsAwesome I forgot to mention that with regards to the styling guide for Python - PEP 8, the standard way have 2 blank lines for functions outside of classes. For functions inside of classes, they only have a single line. As mention here: https://www.python.org/dev/peps/pep-0008/#blank-lines . I do suggest that we adhere to the standard PEP 8 styling rules but if you would prefer we could just have a single blank line instead them.